one-ui/src/test/java/com/marklogic/hub/oneui/TestHelper.java

```diff
@@ -79,6 +79,12 @@ public void authenticateSession() {
        hubConfig.setCredentials(environmentInfo, dataHubDeveloperUsername, dataHubDeveloperPassword);
     }
 
+    public void authenticateSessionAsAdmin() {
+        createUser("admin", "admin","admin");
```

Comment:
It looks like TestHelper already has properties for test.adminUsername and test.adminPassword, so there's no need to do this - the line doesn't really make sense either, as we can safely assume there's an admin user with the admin role.

Also, to mirror similar methods in HubTestBase, call this "runAsAdminUser".

Reply:
@hao1st  Can't this line be removed? There shouldn't be any reason to try to create an admin user.

Reply:
@rjrudin I will fall back it to Environment Manager user instead of user though i did not get why we could not use "admin".